cmake/external_onednn.cmake

```diff
@@ -260,42 +263,6 @@ else()
         )
 endif()
 
-if(WIN32)
```

Comment:
Why remove support for win32?

Reply:
The PR does not remove WIN32 support. The code that is removed is for copying libraries to a new location. The PR passes CMAKE_LIBRARY_OUTPUT_DIRECTORY and CMAKE_ARCHIVE_OUTPUT_DIRECTORY to the external project and the code for copying is not needed anymore.